flake8_dcos_lint/check_rules.py

```diff
@@ -0,0 +1,10 @@
+import re
+
+from collections import namedtuple
+
+CheckRegex = namedtuple("CheckRegex", "regex code reason")
+
+
+regex_rules = [
+    CheckRegex(re.compile(r'assert\s(\w+\.ok)$'), "D001", "Invalid assertion of type assert <symbol>.ok found.")
```

Comment:
So, the rationale is probably this. (I had to figure out myself as I got the rule from hearsay)


```
    @property
    def ok(self):
        try:
            self.raise_for_status()
        except HTTPError:
            return False
        return True
```

* This hides the status_code and it is better to verify the status-code than a boolean that hides the status_code.
* It's not documented in the API (http://docs.python-requests.org/en/master/api/#requests.Response) 

I agree a short-comment, that states the reason for the rule will be a better and will help with buy-in.